examples/lua/nats_adapter.lua

```diff
@@ -0,0 +1,94 @@
+local json = require ("lunajson")
+local nats = require ("nats")
+
+function nats_connect()
+    local nats_params = {
+        host ='0.0.0.0',
```

Comment:
please add host and port as parameters as well here
they should be set as constant in line 67